tools/merge_chainspec_accounts.js

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env node
+// Called like this:
+// node merge_chainspec_accounts.js chainspec.json accounts.json
+const fs = require("fs");
+
+const chainspecPath = process.argv[2];
+const accountsPath = process.argv[3];
+
+const chainspec = JSON.parse(fs.readFileSync(chainspecPath));
+const accounts = JSON.parse(fs.readFileSync(accountsPath));
+
+Object.entries(accounts).forEach(([address, obj]) => {
+  if (chainspec.accounts[address]) {
+    console.warn(`Account at address ${address} already exists.`);
+  }
+  chainspec.accounts[address] = obj;
+});
+
+console.log(JSON.stringify(chainspec, null, 4));
```

Comment:
What is the purpose of this?

Reply:
I don't know enough about JS, but are you using a *logging* mechanism to give feedback to the user on `stdout`? Can I change the behavior of `console.log()` to log somewhere else? E.g. a file? In that case I would prefer explicitly writing to `stdout` and not a log.